app/models/course/assessment.rb

```diff
@@ -18,6 +18,8 @@ class Course::Assessment < ApplicationRecord
   after_commit :grade_with_new_test_cases, on: :update
   before_save :save_tab, on: :create
 
+  enum randomization: { prepared: 0 }
```

Comment:
This differs from the proposal in #3343 ?

Reply:
Er ya we leave this variable as `nil` to indicate no randomization instead. Part of the reason is because "none" is reserved 😂 but it also makes sense that nil randomization means no randomization.

Reply:
`on_demand: 1` will be added when it is implemented